.github/workflows/sonarqube.yml

```diff
@@ -39,4 +39,4 @@ jobs:
 
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-          SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
+          SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
```

Comment:
Can you revert this file to the previous version? I think these change is not needed.